### DISCUSSION TITLE
Setting an info box under the title based on meta information provided

### DISCUSSION THREAD

**Discussion by userbradley** [squidfunk/mkdocs-material#5789] [Q&A] [answered]
I am trying to configure something for an internal site where we provide the below on a file

```md
---
title: Page name
jira: abc-123
---

# Page name

## Header 2

Content
```

And on the page it will render an Admonitions (Which I will make in HTML) below the title of the page.

So far for testing I have the below which renders it above the title

```jinja
{% extends "base.html" %}

{% block content %}

<h1> test</h1>
<!-- Add scripts that need to run before here -->
{{ page.content }}
<!-- Add scripts that need to run afterwards here -->
{% endblock %}

```

No matter what I try I cant seem to get this working in the right location.

I've set something similar up before where it notifies you if the page has expired, but it too struggles from the same issue

![image](https://github.com/squidfunk/mkdocs-material/assets/41597815/21666bb4-6874-40fd-a74c-4ae2898d7f2f)

Any help would be appreciated.

**Comment by userbradley** [comment]
@squidfunk Sorry to be really annoying and tag you here, but do you have any idea on how I can achieve this?

I want it to look like the below

![image](https://github.com/squidfunk/mkdocs-material/assets/41597815/161c03d5-fcb5-4efc-9ea3-cf845d5e9d2b)

I've tried following the documentation and using chatGPT but had no luck. I can work out the HTML etc to put there, just need to know how to target the space between the title and the start of the document

**Reply by squidfunk** [reply, marked as answer]
You would have to omit using `h1` titles in your documents, because then you can put your code / custom logic [here](https://github.com/squidfunk/mkdocs-material/blob/382101d8a37c72b44211232885c0fd5367c574af/src/partials/content.html#L38). If the title is included in the Markdown file, there's no way to achieve this using templates, because the `h1` is part of the content.

**Reply by userbradley** [reply]
Thank you so much! <3
